Core/Utilities/InstallTools.py

```diff
@@ -1803,6 +1803,29 @@ def installComponent( componentType, system, component, extensions, componentMod
 
   gLogger.notice( 'Installing %s %s/%s' % ( componentType, system, component ) )
 
+  # Retrieve bash variables to be set
+  result = gConfig.getOption( 'DIRAC/Setups/%s/%s' % ( CSGlobals.getSetup(), system ) )
+  if not result[ 'OK' ]:
+    return result
+  instance = result[ 'Value' ]
+
+  result = getComponentCfg( componentType, system, component, instance, extensions )
+  if not result[ 'OK' ]:
+    return result
+  compCfg = result[ 'Value' ]
+
+  result = _getSectionName( componentType )
+  if not result[ 'OK' ]:
+    return result
+  section = result[ 'Value' ]
+
+  bashVars = ''
+  if compCfg.isSection( 'Systems/%s/%s/%s/%s/BashVariables' % ( system, instance, section, component ) ):
```

Comment:
I would rather call the section "Environment". The bash shell is chosen by the run script itself, no need to stress it here